test/run_test.sh

```diff
@@ -64,6 +64,9 @@ $PYCMD test_cuda.py $@
 echo "Running NCCL tests"
 $PYCMD test_nccl.py $@
 
+echo "Running C++ Extensions tests"
+$PYCMD test_cpp_extensions.py $@
```

Comment:
Shouldn't this run `setup.py` to build the extension? The test will fail otherwise

Reply:
Yes, thanks! Missed this because I already had it installed.